runbotics/runbotics-orchestrator-ui/src/src-app/views/credentials/GridView/CredentialCollectionsGridView/CredentialCollectionsGridView.tsx

```diff
@@ -0,0 +1,150 @@
+    pageSizeFromUrl;
+    const debouncedValue = useDebounce<string>(searchValue.trim(), DEBOUNCE_TIME);
+    const pageSize = pageSizeFromUrl ? parseInt(pageSizeFromUrl) : DEFAULT_COLLECTION_PAGE_SIZE;
+
+    const [currentCollection, setCurrentCollection] = useState(null);
+    const [isEditDialogOpen, setIsEditDialogOpen] = useState(false);
+    const [isDeleteDialogOpen, setIsDeleteDialogOpen] = useState(false);
+
+    useEffect(() => {
+        const pageNotAvailable = allCredentialCollectionsByPage && page >= allCredentialCollectionsByPage.totalPages;
+
+        if (pageNotAvailable) {
+            setPage(0);
+            replaceQueryParams({ page: 0, pageSize, searchValue });
+        } else {
+            replaceQueryParams({ page, pageSize, searchValue });
+        }
+    }, [allCredentialCollectionsByPage]);
+
+    useEffect(() => {
+        dispatch(credentialCollectionsActions.fetchAllCredentialCollectionsByPage({ pageParams: {
+            page,
+            size: pageSize,
+            filter: {
+                contains: {
+                    name: debouncedValue,
+                },
+            }
+        }})).catch(error => {
+            enqueueSnackbar(error.message, { variant: 'error' });
+        }).finally(() => {
+            setIsLoading(false);
+        });
+    }, [page, pageSize, debouncedValue]);
+
+    const handleOpenEditDialog = (id: string) => {
+        setCurrentCollection(credentialCollections.find(collection => collection.id === id));
+        setIsEditDialogOpen(true);
+    };
+
+    const handleOpenDeleteDialog = (id: string) => {
+        setCurrentCollection(credentialCollections.find(collection => collection.id === id));
+        setIsDeleteDialogOpen(true);
+    };
+
+    const collectionTiles = credentialCollections
+        ? [...credentialCollections]
+            .sort((a, b) => b.createdAt.localeCompare(a.createdAt))
+            .map(collection => (
+                <CredentialsCollectionTile
+                    key={collection.id}
+                    collection={collection}
+                    handleOpenEditDialog={handleOpenEditDialog}
+                    handleOpenDeleteDialog={handleOpenDeleteDialog}
+                    setCurrentDialogCollection={setCurrentCollection}
+                />
+            ))
+        : [];
+
+    return (
+        <InternalPage title={translate('Credentials.Collections.Page.Title')}>
+            <Header />
+            <If condition={!isLoading} else={<LoadingScreen />}>
+                <CredentialsHeader
+                    credentialCount={allCredentialCollectionsByPage?.totalElements}
+                    tabName={CredentialsTabs.COLLECTIONS}
+                    items={credentialCollections}
+                    setSearchValue={setSearchValue}
+                    searchValue={searchValue}
+                    sharedWithNumber={null}
+                />
+                {collectionTiles.length > 0 ? (
+                    <>
```

Comment:
Looks unnecessary

Reply:
No, it is not. There is : after that which is displayed when there are no collections yet (fetched, but there is an empty array), the "No credential collection yet" information is being displayed.

Reply:
I meant the \< \>

Reply:
and \</\>